src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -2625,5 +2625,10 @@ public interface IAsyncDisposable
     }
 }
 ";
+
+        internal static string ExpectedOutput(string output)
```

Comment:
>ExpectedOutput [](http://example.com/codeflow?start=31&length=14)

The name doesn't reflect the purpose of the method, therefore, I do not think it should be used for a global/shared helper.